src/gui/guiFormSpecMenu.cpp

```diff
@@ -122,6 +123,10 @@ GUIFormSpecMenu::~GUIFormSpecMenu()
 		table_it.second->drop();
 	}
 
+	for (auto &scrollbar_it : m_scrollbars) {
+		scrollbar_it.second->drop();
+	}
```

Comment:
Please also drop in `regenerateGui` (see line 2466).
(And the curly braces aren't needed.)